CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
   - `KubePdbNotEnoughHealthyPods`
   - `KubeNodePressure`
   - `KubeNodeEviction`
+- []() Allow cluster-admins to opt-into optional monitoring using the `OptionalMonitoring` capability.
```

Comment:
I realize that adding the annotation to the manifests under the `assets/` directory will have no direct effect since there's no logic in CMO to deploy these resources conditionally, right?